client/login.html

```diff
@@ -16,12 +16,17 @@ <h3 class="logo">Entrepre<span>neurship</span></h3>
       </ul>
     </nav>
     <main>
-      <form id="login-form">
-          <div class="logo-login"><img src="./assets/logo.png" alt="logo"></div>
-          <input type="text" name="matric-no" placeholder="Matric NO" required><br>
-          <input type="password" name="password" placeholder="Password" required><br>
-          <button type="submit" name="Login">Login</button>
-      </form>
+      <div id="container">
+        <div id="header">AUL Forum Login</div>
+        <div id="form-holder">
+          <form id="login-form">
+            <input name="matric-no" type="text" placeholder="Matric Number">
+            <input name="password" type="password" placeholder="Password">
+            <button type="submit">LOG IN</button>
+            <button>SIGN UP</button>
```

Comment:
why is there is a sign up button in login page?

Reply:
standard login forms have a sign up button beneath the login button, the package.json was accidentally removed, dont merge it yet anyways, ill make another updated pull request

Reply:
the sign up button would redirect to the register page

Reply:
ok cool.. also add `yarn-error.log` to gitignore